modules/metastorage-cache/src/main/java/org/apache/ignite/internal/metastorage/cache/IdempotentCacheVacuumizer.java

```diff
@@ -113,8 +112,9 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
                 () -> {
                     if (triggerVacuumization.get()) {
                         try {
-                            vacuumizationAction.accept(hybridTimestamp(clockService.nowLong()
-                                    - (idempotentCacheTtl.value() + clockService.maxClockSkewMillis())));
+                            vacuumizationAction.accept(clockService.now()
```

Comment:
Could you please extract the time calculation into an intermediate variable? It would make the code easier to read.